packages/fast-components-react-msft/assets/carousel-hero-content.tsx

```diff
@@ -1,18 +1,16 @@
-import React from "react";
+import React, { Fragment } from "react";
 import {
+    Button,
```

Comment:
nit: Can we use something other than button here? If it's "accent", then lets just use `<AccentButton />`